media-libs/libgroove/Manifest

```diff
@@ -0,0 +1,3 @@
+DIST libgroove-4.3.0.tar.gz 48857 SHA256 76f68896f078a9613d420339ef887ca8293884ad2cd0fbc031d89a6af2993636 SHA512 30d75a3b1b3efa9b33687bb146648b66f8f4c718551c0db2c24c0259f09cf355a7c7bbac40d60d7a97bbbd7f83401d09464c6ac2b18cdfcd6198220cefc8a705 WHIRLPOOL 4040769723e7891b285318d538cf628203289aca7f7a655855716fe450f58712cd64b2fdc2abbd316bef6f80bf6b65ae6620b9ac339b3ebd5c7e091a36129040
+EBUILD libgroove-4.3.0.ebuild 794 SHA256 0074fb7f431bc884a75449c5927b2d96a1d6b176fa6ceeb8273fe0f5c6720737 SHA512 cbffbb8b868a43989056199c31c0a3a2cb6d1da51b96d920b60ba8590409d32fcde635d00031dfc7a6f6089898f9c056e8bdae41e088c8ac5094515de3ca3bc6 WHIRLPOOL 7bc70a6373f690b1b3b8886ace7c099c6fc42b100202211dad733b8d5a36376b6ac6e1ca4ec7cf65d6207280059d7dd64b41d9a71d759858c3a99d24ebe8a31b
```

Comment:
Do you run `repoman manifest` in the local clone of Gentoo repository, with `metadata/layout.conf` intact? What version of Porage do you have installed?